src/coreComponents/constitutive/ExponentialRelation.hpp

```diff
@@ -421,19 +421,19 @@ class ExponentialRelation
 private:
 
   /// variable shift
-  T m_x0;
+  T m_x0{};
```

Comment:
there is a bit of a structural problem here. We have constructors, but none of these variables are initialized there. They are set in functions called from within the constructor....but that is a different thing.

Reply:
so how should it be?

Reply:
@rrsettgast please let me know what to fix?